tests/test_unasync.py

```diff
@@ -2,9 +2,9 @@
 import os
 import shutil
 import subprocess
-import tempfile
 
 import pytest
+from backports import tempfile
```

Comment:
I think we should add a comment here why we are using backports.tempfile

Reply:
Thanks, done.